### DISCUSSION TITLE
GitHub student developer pack issues

### DISCUSSION THREAD

**Discussion by Matthew-Ayinde** [community/community#133665] [GitHub Education]
### Select Topic Area

Question

### GitHub Feature Area

GitHub Education

### Body

Hi! I have been verified as a student for GitHub education. However I am not able to access the benefits that come with it. How do I go about it? 🙏

**Comment by romhenri** [comment]
Have you verified at current account?

**Reply by Matthew-Ayinde** [reply]
Yes I have. But I still don't know how to access those benefits. I was hoping a link would be sent or something like that after verification?

**Reply by romhenri** [reply]
If you have GitHub Pro, it must be working by now.

What are you interested in?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Matthew-Ayinde ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by SohanKafle** [comment]
Hi [Matthew-Ayinde](https://github.com/Matthew-Ayinde),

If you're verified for GitHub Education but can't access the benefits, try these steps:

1. Confirm Verification: Make sure your student status is confirmed on your GitHub Education account.
2. Log In: Ensure you're logged in with the account that was verified as a student.
3. Check Pack Activation: Go to [GitHub Education](https://education.github.com/pack) and confirm that the Student Developer Pack is activated for your account.
4. Contact Support: If the issue persists, contact [GitHub Support](https://support.github.com/) for assistance.

**Comment by davevad93** [comment]
Hi @Matthew-Ayinde ,

Does the status of your application indicate the following?:

-  `Congrats! Your request for @Matthew-Ayinde was approved and your benefits are now available.`

If so you should have access to your benefits, so first try going to the [education learner page](https://education.github.com/learner/learn) and check for claimed offers as in the screenshot below:

![0](https://github.com/user-attachments/assets/79ac97a5-b215-486c-8fd0-370805bd05f7)

Next, try going [here](https://education.github.com/pack/redeem/copilot-student) to access your free `Copilot` subscription. If you can't access any of the `Student Developer Pack` benefits, you should contact support by opening a ticket [here](https://support.github.com/contact/education), select `I'm a verified student and I'm having trouble redeeming my Global Campus offers.` and explain your issue.

**Reply by Matthew-Ayinde** [reply]
Thank you dave. I've resolved it
